readme.md

```diff
@@ -1,4 +1,4 @@
-# Lighthouse  [![Linux Build Status](https://img.shields.io/travis/GoogleChrome/lighthouse/master.svg)](https://travis-ci.org/GoogleChrome/lighthouse) [![Windows Build Status](https://ci.appveyor.com/api/projects/status/46a5yoqc3hk59bl5/branch/master?svg=true)](https://ci.appveyor.com/project/paulirish/lighthouse/branch/master) [![Coverage Status](https://img.shields.io/coveralls/GoogleChrome/lighthouse/master.svg)](https://coveralls.io/github/GoogleChrome/lighthouse?branch=master) [![NPM lighthouse package](https://img.shields.io/npm/v/lighthouse.svg)](https://npmjs.org/package/lighthouse)
+# Lighthouse  [![GitHub Actions Status Badge](https://github.com/GoogleChrome/lighthouse/workflows/💡🏠/badge.svg)](https://github.com/GoogleChrome/lighthouse/actions) [![Linux Build Status](https://img.shields.io/travis/GoogleChrome/lighthouse/master.svg)](https://travis-ci.org/GoogleChrome/lighthouse) [![Coverage Status](https://img.shields.io/coveralls/GoogleChrome/lighthouse/master.svg)](https://coveralls.io/github/GoogleChrome/lighthouse?branch=master) [![NPM lighthouse package](https://img.shields.io/npm/v/lighthouse.svg)](https://npmjs.org/package/lighthouse)
```

Comment:
> `(https://github.com/GoogleChrome/lighthouse/workflows/💡🏠/badge.svg)](https://github.com/GoogleChrome/lighthouse/actions)`

Yay, but this is where we start to regret our workflow name. They still haven't fixed handling of emoji in their badge generator 😢 

<img width="85" alt="bad badge screenshot" src="https://user-images.githubusercontent.com/316891/88727634-fc837b80-d0fd-11ea-9227-0750c7ef4ca9.png">

I guess we can pretend that little bit of the lightbulb peeking out isn't there and it's just a nameless badge until they fix things :)

Reply:
> I guess we can pretend that little bit of the lightbulb peeking out isn't there and it's just a nameless badge until they fix things

LOL I didn't even notice the peeking lighthouse I just thought "huh that's weird they don't give it a name" 😆